.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
We normally don't checkin editor-specific files. You can add the whole `.idea` folder to `.gitignore`.

Reply:
The `.DS_Store` files should not be checked in either.

Reply:
Sure, done.